README.md

```diff
@@ -35,6 +39,11 @@ nano ~/.bashrc
 export OPENAI_API_KEY=<your_key_here>
 ```
 
+After that, to make the changes to the `.bashrc` file take effect without opening another terminal,
+you can run:
+
+    source ~/.bashrc
```

Comment:
I think it makes sense to use code block here for consistency with the rest of the file

Reply:
Hello

Ok, just to be sure, you mean a ```bash``` block right?

Reply:
yes